Mini-Project.ipynb

```diff
@@ -41,16 +41,136 @@
+      "text/plain": [
+       "<Figure size 1200x600 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "caffeine_counts = pd.crosstab(df['caffeineIntake'], df['classStanding'])\n",
+    "\n",
+    "caffeine_counts = caffeine_counts.loc[caffeine_counts.sum(axis=1).sort_values().index]\n",
+    "\n",
+    "caffeine_counts.plot(kind='bar', stacked=True, figsize=(12, 6), color=['skyblue', 'salmon', 'lightgreen', 'orange', 'red', 'magenta'])\n",
```

Comment:
Change to leave variables in proper order, from least amount, to greatest amount